analyzers/Abuse_Finder/requirements.txt

```diff
@@ -1,3 +1,4 @@
 cortexutils
+git+https://github.com/monoidic/ipwhois.git
```

Comment:
@vpiserchia What's the need for this dependency?

Reply:
By using this forked version of the library a newer version of the `dnspython` (v2+) lib is installed. This way all analyzers dependencies are harmonized and can be installed in the same docker or system.

Reply:
Hello! I'm removing it from requirements as package abuse_finder already installs dnspython v2+ (specifically, v2.7.0). It also installs the official ipwhois v1.3.0, which is current latest version; no issues observed with the analyzer on my end.